src_c/event.c

```diff
@@ -1591,6 +1591,41 @@ get_grab(PyObject *self, PyObject *_null)
     return PyBool_FromLong(mode);
 }
 
+static PyObject *
+set_keyboard_grab(PyObject *self, PyObject *arg)
+{
+    int doit = PyObject_IsTrue(arg);
+    if (doit == -1)
+        return NULL;
+
+    VIDEO_INIT_CHECK();
+
+    SDL_Window *win = pg_GetDefaultWindow();
+    if (win) {
+        if (doit) {
+            SDL_SetWindowKeyboardGrab(win, SDL_TRUE);
```

Comment:
There needs to be a guard around SDL_SetWindowKeyboardGrab and SDL_GetWindowKeyboardGrab in case it's not available in older versions of SDL. When it's not available just do nothing I think.

```c
#if SDL_VERSION_ATLEAST(2, 0, 14)
    SDL_Window *win = pg_GetDefaultWindow();
    if (win) {
    ...
#endif
Py_RETURN_NONE;
```